common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -601,6 +608,85 @@ def test_tooltip(self):
         self.assertTrue(self.courseware_page.tooltips_displayed())
 
 
+class PreRequisiteCourseTest(UniqueCourseTest):
+    """
+    Tests that pre-requisite course messages are displayed
+    """
+
+    def setUp(self):
+        """
+        Initialize pages and install a course fixture.
+        """
+        super(PreRequisiteCourseTest, self).setUp()
+
+        CourseFixture(
+            self.course_info['org'], self.course_info['number'],
+            self.course_info['run'], self.course_info['display_name']
+        ).install()
+
+        self.prc_info = {
+            'org': 'test_org',
+            'number': self.unique_id,
+            'run': 'prc_test_run',
+            'display_name': 'PR Test Course' + self.unique_id
+        }
+
+        CourseFixture(
+            self.prc_info['org'], self.prc_info['number'],
+            self.prc_info['run'], self.prc_info['display_name']
+        ).install()
+
+        pre_requisite_course_key = generate_course_key(
+            self.prc_info['org'],
+            self.prc_info['number'],
+            self.prc_info['run']
+        )
+        self.pre_requisite_course_id = unicode(pre_requisite_course_key)
+
+        self.dashboard_page = DashboardPage(self.browser)
+        self.settings_page = SettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+
+        )
+        # Auto-auth register for the course
+        AutoAuthPage(self.browser, course_id=self.course_id).visit()
+
+    def test_dashaboad_message(self):
+        """
+        Verify that pre-requisite messages are displayed.
```

Comment:
A best practice for bok choy tests is to include a BDD description of the test in the comments. For example, 
https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L40